### PR TITLE
Microsoft.Identity.Web.UI optional steps

### PR DESCRIPTION
**Pull request by TiagoBrenck**
## Purpose
Added optional step on first samples, explaining how to setup Microsoft.Identity.Web.UI

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: 1-WebApp-OIDC/1-1-MyOrg/README.md [hunk 1]
```diff
@@ -172,13 +172,34 @@ cd "1-WebApp-OIDC\1-1-MyOrg"
     - update the `sslPort` of the `iisSettings` section to be `44321`
     - in the `applicationUrl` property of use `https://localhost:44321`
 
+1. (Optional) If you don't have a custom `AccountController` to handle the *sign-in* and *sign-out* requests, you can use `Microsoft.Identity.Web.UI` built-in one. For that, please include this change on **Startup.cs**:
```
Comment (jennyf19) [inline 1-WebApp-OIDC/1-1-MyOrg/README.md:175]: nit: add "the" between "you can use" and "Microsoft.Identity.web U..."

Comment (TiagoBrenck) [reply 1-WebApp-OIDC/1-1-MyOrg/README.md]: Fixed. Thanks Jenny

File: 1-WebApp-OIDC/1-1-MyOrg/README.md [hunk 2]
```diff
@@ -172,13 +172,34 @@ cd "1-WebApp-OIDC\1-1-MyOrg"
     - update the `sslPort` of the `iisSettings` section to be `44321`
     - in the `applicationUrl` property of use `https://localhost:44321`
 
+1. (Optional) If you don't have a custom `AccountController` to handle the *sign-in* and *sign-out* requests, you can use the `Microsoft.Identity.Web.UI` built-in one. For that, please include this change on **Startup.cs**:
```
Comment (pmaytak) [inline 1-WebApp-OIDC/1-1-MyOrg/README.md:175]: nit: `change on` > `change in`?

File: 1-WebApp-OIDC/1-1-MyOrg/README.md [hunk 3]
```diff
@@ -172,13 +172,34 @@ cd "1-WebApp-OIDC\1-1-MyOrg"
...
+
+    - at the top of the file, add the following using directive:
+
+      ```CSharp
+        using Microsoft.Identity.Web.UI;
+      ```
+    - in the `ConfigureServices` method, change the **AddControllersWithView** code snippet with this:
```
Comment (pmaytak) [inline 1-WebApp-OIDC/1-1-MyOrg/README.md:182]: nit: `with this` > `to this`

File: 1-WebApp-OIDC/1-1-MyOrg/README.md [hunk 4]
```diff
@@ -172,13 +172,34 @@ cd "1-WebApp-OIDC\1-1-MyOrg"
...
+        services.AddControllersWithViews(options =>
+        {
+            var policy = new AuthorizationPolicyBuilder()
+                .RequireAuthenticatedUser()
+                .Build();
+            options.Filters.Add(new AuthorizeFilter(policy));
+        }).AddMicrosoftIdentityUI();
+      ```
+
+    - on **Views/Shared/_LoginPartial.cshtml**, change the **asp-area** tag to: `asp-area="MicrosoftIdentity"`
```
Comment (pmaytak) [inline 1-WebApp-OIDC/1-1-MyOrg/README.md:194]: nit: `on` > `in`

**Pull request merged**